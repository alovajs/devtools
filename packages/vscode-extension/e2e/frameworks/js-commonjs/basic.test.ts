import { expect, getExt, Global, is, not, openFile, setupTest, timeout } from '../../ctx'

setupTest('js-commonjs', () => {
  it('opens entry file', async () => {
    await openFile('package.json')
  })

  it('is active', () => {
    const ext = getExt()
    is(ext?.isActive, true)
  })
  it('should works', async () => {
    await timeout(500)
    not(Global, undefined)
    expect(1 + 1).to.matchSnapshot()
  })
})
