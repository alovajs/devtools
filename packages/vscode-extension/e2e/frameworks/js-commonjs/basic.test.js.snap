// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`js-commonjs should works 1`] = `2`;
